### PR TITLE
chore: Bump DC/OS UI v2.83.0

### PR DESCRIPTION
**Pull request by TattdCodeMonkey**
## High-level description

What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [DCOS-60342](https://jira.mesosphere.com/browse/DCOS-60342)
  - [DCOS-60344](https://jira.mesosphere.com/browse/DCOS-60344)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by nLight** [comment]
@mesosphere-mergebot label [Ready For Review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines). Thank you!
 
 ## DC/OS 1.13.6
+## DC/OS 1.13.6 (in development)
```
Comment (nLight) [inline CHANGES.md]: let's not change that

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+
+* Updated DC/OS UI to [1.13+v2.83.0](https://github.com/dcos/dcos-ui/releases/tag/1.13+v2.83.0).
+* Updated DC/OS UI to [1.13+v2.83.1](https://github.com/dcos/dcos-ui/releases/tag/1.13+v2.83.1).
```
Comment (nLight) [inline CHANGES.md]: let's keep only 
```
* Updated DC/OS UI to [1.13+v2.83.1](https://github.com/dcos/dcos-ui/releases/tag/1.13+v2.83.1).
```

File: CHANGES.md (hunk 1 above)
Comment (nLight) [reply CHANGES.md]: @Poltergeist

**Comment by orsenthil** [comment]
@mesosphere-mergebot merge-it

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Pull request merged**